utils/Extract_stimulus_delay_statistics.ipynb

```diff
@@ -0,0 +1,356 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Extraction of stimulus delay data.\n",
+    "In this notebook you will be able to find the delay of audio and video. The two delays will be shown on different figures as histograms.\n",
+    "\n",
+    "## how?\n",
+    "### Visual delay\n",
+    "1. First the right methods will be imported.\n",
+    "    * Here a method called \"extract_delays\" from the file \"extract_delays_meg\" from folder \"meeg\" in the path below\n",
+    "2. The visual trigger code is set\n",
+    "3. Then the raw file with the collected data, which is delayed, will be set.\n",
+    "4. When using the methoed, the misc_channel is set to MISC001, as standard, which is for the diode output.\n",
+    "    * This diode output detects every time the screen changes.\n",
+    "5. When done, it will show 2 figures for the delay of visual stimulation.\n",
+    "\n",
+    "### Audio delay\n",
+    "1. The trigger code gets changed, so the right trigger data is used\n",
+    "2. The misc_channel is set to MISC002. This is a copy of the soundcard output from the STIMpc.\n",
+    "3. When done, it will show 2 figures for the delay of audio stimulation.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "%matplotlib qt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### kommentar\n",
+    "Filstien nedunder behøver ikke at være særlig specific, da der kun findes 1 fil der hedder extract_delays_MEG\n",
+    "Det virker hvis man sletter /meeg-python og de sidste to /git/meeg-python"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "sys.path.insert(0, '/Users/matimero/src/git/meeg-python')"
```

Comment:
ikke sti til din egen user-folder. Jeg sørger for at `meeg-python` master er tilgænglig for alle. `sys.path.insert` burde ikke være nødvendig hvis ikke man vil bruge sine egne versioner.